test/app/stats_test.rb

```diff
@@ -0,0 +1,58 @@
+require_relative '../app_helper'
+require 'mocha/setup'
+
+class StatsTest < Minitest::Test
+  include Rack::Test::Methods
+  include AppTestHelper
+  include DBCleaner
+
+  def app
+    ExercismWeb::App
+  end
+
+  def alice_attributes
+    {
+      username: 'alice',
+      github_id: 1,
+      email: "alice@example.com",
+      created_at: 1.year.ago,
+    }
+  end
+
+  def opted_out_attributes
+    {
+      motivation_experiment_opt_out: true,
+      username: 'opted_out',
+      github_id: 2,
+      email: "opted_out@example.com",
+      created_at: 1.year.ago,
+    }
+  end
+
+  attr_reader :alice, :opted_out
+  def setup
+    super
```

Comment:
`DBCleaner` starts a transaction in the superclass. When overriding `setup`, not calling `super` means that the transaction doesn't start, and any records created get left in the database after each test run.